pkgs/development/compilers/ocaml/5.0.nix

```diff
@@ -0,0 +1,9 @@
+import ./generic.nix {
+  major_version = "5";
+  minor_version = "0";
+  patch_version = "0-beta1";
+  src = fetchTarball {
+    url = "https://caml.inria.fr/pub/distrib/ocaml-5.0/ocaml-5.0.0~beta1.tar.xz";
```

Comment:
Don't we prefer github url ?

Reply:
Ok now it seem we prefer inria url for ocaml:
https://github.com/NixOS/nixpkgs/blob/af34e3be8c7d11b89779ccb5617c0c5f843dda6b/pkgs/development/compilers/ocaml/generic.nix#L23